includes/RestApi/IntegrationsController.php

```diff
@@ -138,4 +150,17 @@ public function get_razorpay_status() {
 			200
 		);
 	}
+
+	public function get_hiive_token() {
+		$hiive_token = HiiveConnection::get_auth_token();
+
+			if ( ! $hiive_token ) {}
+		return new \WP_REST_Response(
+			array(
+				'token' => $hiive_token ? $hiive_token : 'test2',
```

Comment:
We should probably just return an empty string instead of `test2`?

Reply:
We're storing it as ``Key value pair in cloudflare`` and as key can't be blank. In local, as we don't have a ``hiiive_token`` .So, we've added a placeholder variable as ``test2``